server/webapp/WEB-INF/rails/webpack/views/pages/new_plugins/spec/plugins_widget_spec.tsx

```diff
@@ -16,10 +16,10 @@
 
 import * as m from "mithril";
 import {PluginInfo} from "models/shared/plugin_infos_new/plugin_info";
-import * as styles from "../index.scss";
 import {PluginsWidget} from "../plugins_widget";
 
 import * as collapsiblePanelStyles from "views/components/collapsible_panel/index.scss";
+import * as headerIconStyles from "views/components/header_icon/index.scss";
```

Comment:
No file with the above mentioned path exists. Yarn build is broken because of the above changeset

Reply:
I couldn't get the build to work due to unexisting `header_icon` component.